code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -1,4 +1,130 @@
+	id = /datum/reagent/medicine/calomel
+	results = list(/datum/reagent/medicine/calomel = 1)
+	required_reagents = list(/datum/reagent/consumable/tea = 1, /datum/reagent/medicine/C2/multiver = 1)
+	required_temp = 374
+
+/datum/chemical_reaction/potass_iodide //technically rads but rads do tox so *wink*
+	name = "Potassium Iodide"
+	id = /datum/reagent/medicine/potass_iodide
+	results = list(/datum/reagent/medicine/potass_iodide = 2)
+	required_reagents = list(/datum/reagent/potassium = 1, /datum/reagent/iodine = 1)
+
+
+/******OXY******/
+
+/datum/chemical_reaction/salbutamol
+	name = "Salbutamol"
+	id = /datum/reagent/medicine/salbutamol
+	results = list(/datum/reagent/medicine/salbutamol = 1)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1,/datum/reagent/medicine/C2/convermol = 1)
+
+/******COMBOS******/
+/datum/chemical_reaction/salglu_solution
+	name = "Saline-Glucose Solution"
+	id = /datum/reagent/medicine/salglu_solution
+	results = list(/datum/reagent/medicine/salglu_solution = 3)
+	required_reagents = list(/datum/reagent/consumable/sodiumchloride = 1, /datum/reagent/water = 1, /datum/reagent/consumable/sugar = 1)
+
+/datum/chemical_reaction/instabitaluri
+	name = "Synthflesh (Instabitaluri)"
+	id = /datum/reagent/medicine/C2/instabitaluri
+	results = list(/datum/reagent/medicine/C2/instabitaluri = 1)
+	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/carbon = 1, /datum/reagent/medicine/C2/libital = 1)
+
+/datum/chemical_reaction/granibitaluri
+	name = "Granibitaluri"
+	id = /datum/reagent/medicine/granibitaluri
+	results = list(/datum/reagent/medicine/granibitaluri = 1)
+	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/space_cleaner/sterilizine = 1) //haha guys totally not some sort of saline reference!
+	required_temp = 373
+
+/datum/chemical_reaction/mine_salve
+	name = "Miner's Salve"
+	id = /datum/reagent/medicine/mine_salve
+	results = list(/datum/reagent/medicine/mine_salve = 3)
+	required_reagents = list(/datum/reagent/oil = 1, /datum/reagent/water = 1, /datum/reagent/iron = 1)
+
+/datum/chemical_reaction/mine_salve2
+	name = "Miner's Salve"
+	id = /datum/reagent/medicine/mine_salve
+	results = list(/datum/reagent/medicine/mine_salve = 15)
+	required_reagents = list(/datum/reagent/toxin/plasma = 5, /datum/reagent/iron = 5, /datum/reagent/consumable/sugar = 1) // A sheet of plasma, a twinkie and a sheet of metal makes four of these
+
+/datum/chemical_reaction/regen_jelly
+	name = "Regenerative Jelly"
+	id = /datum/reagent/medicine/regen_jelly
+	results = list(/datum/reagent/medicine/regen_jelly = 2)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/toxin/slimejelly = 1)
+
+//Crit-based Combos
+/datum/chemical_reaction/atropine
+	name = "Atropine"
+	id = /datum/reagent/medicine/atropine
+	results = list(/datum/reagent/medicine/atropine = 1)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/medicine/epinephrine = 1)
```

Comment:
Oh boy, I definitely won't accidentally cause this reaction to occur in one of my patients (because you're making omnizine a worse healing chem than granibitaluri is (I'm standing by that statement, by the way))!